.env.example

```
@@ -1,8 +1,94 @@
-ELIZAOS_REST_HOSTNAME=localhost
-ELIZAOS_REST_PORT=3000
\ No newline at end of file
+### elizaOS Environment Variables ###
+# To get started, copy this file to .env, or make a .env and add the settings you'd like to override
+# Please read the comments for each of the configurations
+
+## The only thing you ABSOLUTELY NEED to get up and running is one of the model provider keys,
+## i.e. OPENAI_API_KEY or ANTHROPIC_API_KEY, or setup the ollama plugin
+## Everything else is optional, and most settings and secrets can be configured in your agent or through the GUI
+## For multi-agent, each agent will need keys for the various services it is connected to
+-------------------------------
```

dotenv-linter (3.3.0)

[warning] [KeyWithoutValue] The ------------------------------- key should be with a value or have an equal sign

(KeyWithoutValue)